buildroot/share/PlatformIO/variants/BIGTREE_SKR_PRO_1v1/PeripheralPins.c

```diff
@@ -104,7 +110,14 @@ const PinMap PinMap_I2C_SCL[] = {
   {PB_8,  I2C1, STM_PIN_DATA(STM_MODE_AF_OD, GPIO_NOPULL, GPIO_AF4_I2C1)},
   {PB_10, I2C2, STM_PIN_DATA(STM_MODE_AF_OD, GPIO_NOPULL, GPIO_AF4_I2C2)},
   #if STM32F4X_PIN_NUM >= 144  //144 pins mcu, 114 gpio
-    {PF_1,  I2C2, STM_PIN_DATA(STM_MODE_AF_OD, GPIO_NOPULL, GPIO_AF4_I2C2)},
+    ifSTM32F4X_PIN_NUM >=176){
+      //{PF_1,  I2C2, STM_PIN_DATA(STM_MODE_AF_OD, GPIO_NOPULL, GPIO_AF4_I2C2)},
+        {PH_4,  I2C2, STM_PIN_DATA(STM_MODE_AF_OD, GPIO_NOPULL, GPIO_AF4_I2C2)},
+        {PH_7,  I2C3, STM_PIN_DATA(STM_MODE_AF_OD, GPIO_NOPULL, GPIO_AF4_I2C3)},
+    }
+  else{
+        {PF_1,  I2C2, STM_PIN_DATA(STM_MODE_AF_OD, GPIO_NOPULL, GPIO_AF4_I2C2)},
+  }
```

Comment:
Same here.

Reply:
ifSTM32F4X_PIN_NUM

Reply:
> Same here.
This looks like it's going to be written like this on line 113 
if (STM32F4X_PIN_NUM >=176)

Reply:
> Same here.
Well, ok, Well, ok, use preconditioning,
#if (STM32F4X_PIN_NUM >=176){
...
}
 #else {
...
}
#endif